test/mocha.opts

```diff
@@ -0,0 +1,4 @@
+--compilers js:babel-register,jsx:babel-register
+--require airbnb-js-shims
+--require ./test/_helpers/ignoreSVGStrings.jsx
```

Comment:
This fails to include the file that restores sinon stubs; please revert this part and restore `npm run mocha`, and have that pass the file there. `mocha` doesn't work with global before/after handlers if they're `--require`d in.